src/screens/Home.tsx

```diff
@@ -7,31 +8,29 @@ import {
 
 import { useAppDispatch, useAppSelector } from '../app/hooks'
 import { ClearButton } from '../components/ClearButton'
+import { LoadingActivityIndicator } from '../components/Loading'
 import Section from '../components/Section'
-import { clearAllDone } from '../features/todosState'
+import { todosFunctionForStateSelector } from '../entities/constants'
+import { Routes } from '../entities/constants'
+import { clearAllDone, getAllTodos } from '../features/todosState'
 import { HomeNavProps } from '../navigation/types'
 import { Colors } from '../styles/colors'
-
-export interface mockedType {
-  id: number
-  title: string
-  description: string
-  checked: boolean
-}
-
 export default function HomeScreen({
   navigation,
 }: HomeNavProps): React.JSX.Element {
-  const todos = useAppSelector(state => state.todos)
   const dispatch = useAppDispatch()
-
+  useEffect(() => {
+    dispatch(getAllTodos())
+  }, [])
+  const todos = useAppSelector(todosFunctionForStateSelector)
+  const loading = todos.loading
```

Comment:
```suggestion
  const { loading, todos } = useAppSelector(todosFunctionForStateSelector)
```

(in case you want to bring everything from the selector itself. Remember you are subscribing to the whole state)